icacls_work.rb

```diff
@@ -0,0 +1,36 @@
+class Rights
+
+  attr_reader :path, :name
+  def initialize(path, name)
+    @path = path
+    @name = name
+  end
+
+  def get_res_string(name_file)
+    system("cd #{@path} && icacls #{@name} > #{name_file}.txt")
+    written_file = File.open("#{@path}\\#{name_file}.txt")
+    written_file.each do |line|
+      return line if line.include?'Users'
+    end
+  end
+
+  def print_rights(rights_line)
```

Comment:
change wording 'rights' to 'permissions'

Reply:
Replace all words 'rights' with 'permissions'